doc/site/guides/headless-and-dedi.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Headless and dedicated
+parent: Guides
+permalink: guides/headless-and-dedi
+author: Sprung
```

Comment:
```suggestion
sprunk
```

As the name of the author we are using. Feel free to rename the file and content as you wish on `_authors/sprunk.markdown`